.github/workflows/ci.yml

```diff
@@ -272,7 +272,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2-beta
         with:
-          node-version: '12'
+          node-version: '16'
```

Comment:
Why does this specific step need node 16?

Reply:
I noticed it was not synced anymore with the node version used in the Dockerfile, so that's the primary reason. Secondary, newer node versions come with a newer npm version (npm 8) which uses a newer format for package-lock.json, which is used during `npm ci`.